src/display/Table/Table.tsx

```diff
@@ -0,0 +1,168 @@
+import React from "react";
+import {
+  Table as AriaTable,
+  TableProps as AriaTableProps,
+  ColumnProps,
+  ColumnResizer,
+  ResizableTableContainer,
+  RowProps,
+  TableBody,
+} from "react-aria-components";
+import styled, { css } from "styled-components";
+import { body1, theme } from "src/guidelines/theme";
+import { sizing } from "src/utils/utils";
+import TableCell from "./TableCell";
+import TableColumn from "./TableColumn";
+import TableHeader from "./TableHeader";
+import TableRow from "./TableRow";
+
+interface DataColumn extends ColumnProps {
+  isResizable?: boolean;
+  name: string;
+}
+
+interface DataRows<T> extends RowProps<T> {
+  isColored?: boolean;
+}
+
+type TableProps<T> = {
+  dataColumns: Array<DataColumn>;
+  dataRows: Array<DataRows<T>>;
+  tableTitle: string;
+  evenRowsBackgroundIsColored: boolean;
+} & AriaTableProps;
+
+const StyledResizableTableContainer = styled(ResizableTableContainer)`
+  width: 100%;
+  overflow: auto;
+  position: relative;
+  border: 1px solid ${theme.color.gray100};
+  border-radius: ${theme.borderRadius.default};
+  background: ${theme.color.white};
+`;
+
+const StyledTable = styled(AriaTable)`
+  ${body1};
+  width: 100%;
+  table-layout: initial !important;
```

Comment:
can we avoid using **!important** ?

Reply:
I didn't find solution to overload the "table-layout" is setting in the style attribute with js